openshift-examples/keycloak-https-mutual-tls.json

```diff
@@ -55,6 +55,13 @@
             "name": "HOSTNAME_HTTPS",
             "value": "",
             "required": false
+        },
+        {
+            "displayName": "Namespace used for DNS discovery",
+            "description": "This namespace is a part of DNS query sent to Kubernetes API. This query allows the DNS_PING protocol to extract cluster members. This parameter might be removed once https://issues.jboss.org/browse/JGRP-2292 is implemented.",
+            "name": "NAMESPACE",
+            "value": "myproject",
```

Comment:
Can the namespace somehow be templated so this doesn't have to be changed? Surely OpenShift knows what namespace it's being deployed to?

Reply:
Unfortunately no. And as the parameter description mentions, JGRP-2292 prevents us from using a Service name (since it appends `svc.cluster.local` suffix). See the [JGRP-2292](https://issues.jboss.org/browse/JGRP-2292) ticket for details. 

It isn't perfect, I know. But we'll improve it once the JGroups patch is in.

Reply:
This parameter will go away very shortly. We need two JGroups tickets to fix it (JGRP-2295 and JGRP-2292). Rob already sent a pull request for both. So if we are lucky, this parameter will disappear in a couple of weeks.

So what do you think, having the above in mind, should we still enforce this parameter from users? Even though it will be gone shortly?

Reply:
After giving it a little more thought... we can always remove that parameter once all changes are in. Give me a sec to introduce a fix.